CONTRIBUTING.md

```diff
@@ -469,11 +477,11 @@ Note that `Any` is not the correct type to use if you want to indicate
 that some function can accept literally anything: in those cases use
 `object` instead.
```

Comment:
Should we mention `Unused` (for parameters that are unused even in subclassing, and ignored callable return types) with the semantic aliases ?

Reply:
`Unused` equals `object`. Generally speaking, now that Rebecca is moving content into the [Writing Stubs guide](https://typing.readthedocs.io/en/latest/guides/writing_stubs.html), we should review our recommendations here and merge them into to that guide as far as possible.